dynamic_rest/fields/fields.py

```diff
@@ -135,7 +135,7 @@ def bind(self, *args, **kwargs):
                 )
         ):
             self.required = False
-        if 'allow_null' not in self.kwargs and getattr(
+        if 'allow_null' in self.kwargs and getattr(
```

Comment:
Cool, I buy that this is wrong. The problem I'm trying to fix: In the course of moving from the locally-defined get_model_field() in my last PR, to the more complex get_model_field() in meta.py, the unit tests broke. Specifically, a bunch of fields that hadn't been nullable, now became nullable (https://circleci.com/gh/AltSchool/dynamic-rest/534, towards the bottom). I _think_ this is because meta.py's get_model_field is more robust, meaning model_field is more often populated, meaning model_field.null is more often available. So, I wasn't sure if (a) the unit tests are, in fact, wrong, or (b) 'allow_null' was set incorrectly here - as a bug - and that bug was eclipsed by the model_fields that were less populated.

So, I put this back, and change the unit tests? Or are they correct, and the problem is elsewhere?